src/queries/rum-acquisition.sql

```diff
@@ -0,0 +1,368 @@
+      @enddate, # end date
+      @timezone, # timezone
+      'all', # deviceclass
+      @domainkey # domain key to prevent data sharing
+    )
+  WHERE
+    checkpoint IN ('enter', 'utm')
+    AND user_agent NOT IN ('bot', 'undefined')
+),
+
+weekly_events AS (
+  SELECT
+    id,
+    hostname,
+    checkpoint,
+    source,
+    target,
+    weight,
+    TIMESTAMP_TRUNC(time, HOUR, @timezone) AS date,
+    TIMESTAMP_TRUNC(time, ISOWEEK, @timezone) AS dategroup
+  FROM
+    helix_rum.EVENTS_V5(
+      @url, # url
+      CAST(@offset AS INT64), # offset
+      CAST(@interval AS INT64), # days to fetch
+      @startdate, # start date
+      @enddate, # end date
+      @timezone, # timezone
+      'all', # deviceclass
+      @domainkey # domain key to prevent data sharing
+    )
+  WHERE
+    checkpoint IN ('enter', 'utm')
+    AND user_agent NOT IN ('bot', 'undefined')
+),
+
+monthly_events AS (
+  SELECT
+    id,
+    hostname,
+    checkpoint,
+    source,
+    target,
+    weight,
+    TIMESTAMP_TRUNC(time, HOUR, @timezone) AS date,
+    TIMESTAMP_TRUNC(time, MONTH, @timezone) AS dategroup
+  FROM
+    helix_rum.EVENTS_V5(
+      @url, # url
+      CAST(@offset AS INT64), # offset
+      CAST(@interval AS INT64), # days to fetch
+      @startdate, # start date
+      @enddate, # end date
+      @timezone, # timezone
+      'all', # deviceclass
+      @domainkey # domain key to prevent data sharing
+    )
+  WHERE
+    checkpoint IN ('enter', 'utm')
+    AND user_agent NOT IN ('bot', 'undefined')
+),
+
+events AS (
+  SELECT
+    id,
+    date,
+    dategroup,
+    hostname,
+    checkpoint,
+    source,
+    target,
+    weight
+  FROM daily_events
+  WHERE CAST(@granularity AS INT64) = 1
+  UNION ALL
+  SELECT
+    id,
+    date,
+    dategroup,
+    hostname,
+    checkpoint,
+    source,
+    target,
+    weight
+  FROM weekly_events
+  WHERE CAST(@granularity AS INT64) = 7
+  UNION ALL
+  SELECT
+    id,
+    date,
+    dategroup,
+    hostname,
+    checkpoint,
+    source,
+    target,
+    weight
+  FROM monthly_events
+  WHERE CAST(@granularity AS INT64) = 30
+),
+
+enter_events AS (
+  SELECT
+    id,
+    date,
+    dategroup,
+    hostname,
+    checkpoint,
+    source,
+    weight
+  FROM events
+  WHERE
+    checkpoint = 'enter'
+),
+
+utm_source_events AS (
+  SELECT
+    id,
+    date,
+    dategroup,
+    hostname,
+    checkpoint,
+    source,
+    target,
+    weight
+  FROM events
+  WHERE
+    checkpoint = 'utm'
+    AND source = 'utm_source'
+),
+
+utm_medium_events AS (
+  SELECT
+    id,
+    date,
+    dategroup,
+    hostname,
+    checkpoint,
+    source,
+    target,
+    weight
+  FROM events
+  WHERE
+    checkpoint = 'utm'
+    AND source = 'utm_medium'
+),
+
+utm_campaign_events AS (
+  SELECT
+    id,
+    date,
+    dategroup,
+    hostname,
+    checkpoint,
+    source,
+    target,
+    weight
+  FROM events
+  WHERE
+    checkpoint = 'utm'
+    AND source = 'utm_campaign'
+),
+
+utm_term_events AS (
```

Comment:
After discussing with @davidnuescheler and @meryllblanchet, we think capturing the `utm_term` could potentially leak PII since these are direct search terms entered by a user.
We should probably remove it from the query for now, and I'll also exclude it from the RUM enhancer so it's not captured.

Reply:
Done.